crates/ruff_python_formatter/src/expression/expr_lambda.rs

```diff
@@ -38,21 +35,7 @@ impl FormatNodeRule<ExprLambda> for FormatExprLambda {
             )?;
         }
 
-        write!(
-            f,
-            [
-                text(":"),
-                space(),
-                body.format(),
-                // It's possible for some `Arguments` of `lambda`s to be assigned dangling comments.
-                //
-                // a = (
-                //     lambda  # Dangling
-                //     : 1
-                // )
-                dangling_node_comments(parameters.as_ref())
-            ]
-        )
+        write!(f, [text(":"), space(), body.format()])
     }
 
     fn fmt_dangling_comments(
```

Comment:
<i class='graphite__hidden'>[Re: lines 46 to 46]<br/></i>

This comment is no longer true.<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/astral-sh/ruff/6669?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Hmm, it's pretty annoying that you have to go to graphite to see the commented lines.